From 46b013d60d50268470728130f39df11c9921d23a Mon Sep 17 00:00:00 2001
From: ejaaskel <esa.jaaskela@suomi24.fi>
Date: Mon, 11 Sep 2023 16:42:52 +0000
Subject: [PATCH 1/1] Change the default zone

Change the default zone to example_zone. This will set the desired zone
right at the start up.
---
 config/firewalld.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/firewalld.conf b/config/firewalld.conf
index 99d573d..c399b7f 100644
--- a/config/firewalld.conf
+++ b/config/firewalld.conf
@@ -3,7 +3,7 @@
 # default zone
 # The default zone used if an empty zone string is used.
 # Default: public
-DefaultZone=public
+DefaultZone=example_zone
 
 # Clean up on exit
 # If set to no or false the firewall configuration will not get cleaned up
-- 
2.34.1

